lib/vmpooler/providers/vsphere.rb

```diff
@@ -365,15 +353,22 @@ def get_target_datacenter_from_config(pool_name)
           nil
         end
 
-        def generate_vm_hash(vm_object, template_name, pool_name)
-          hash = { 'name' => nil, 'hostname' => nil, 'template' => nil, 'poolname' => nil, 'boottime' => nil, 'powerstate' => nil }
+        def generate_vm_hash(vm_object, pool_name)
```

Comment:
This seems like a good opportunity to add documentation to this method.

Reply:
I added some brief documentation to this method. Let me know if you think it doesn't cover what you were hoping to see, it's pretty brief.

Reply:
👍